aea/helpers/transaction/base.py

```diff
@@ -0,0 +1,526 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
```

Comment:
This class now contains framework side objects for a lot of the ledger and crypto related interactions